buildSrc/src/main/groovy/corda.api-scanner.gradle

```diff
@@ -0,0 +1,4 @@
+// Apply api-scanner
+if (System.getenv('CORDA_ARTIFACTORY_USERNAME') != null || project.hasProperty('cordaArtifactoryUsername')) {
+    project.pluginManager.apply('net.corda.internal.gradle.plugins.api-scanner')
```

Comment:
Question maybe for @ronanbrowne .

I'm guessing this means Open Source users won't be able to run this, which is probably fine, but will this result in an error if they try to do a normal "build"?

Reply:
This won't prevent normal builds, only the api export, diff, and cement

Reply:
@driessamyn a normal build `./gradlew build` or any other task wont trigger the above, only issue is if a non-r3 user tries to use any of the tasks provided by the plugin directly, which as this is a internal tool, my presumption is we don't expect them to.

So the code here ensures we can compile for OS users